maav.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+CMD=$1
+ARG=$2
+
+# TODO: read this from ~/.maavrc
+MAAV_DIR=/home/rpash/maav
```

Comment:
There is a TODO here

Reply:
This script is not done yet

Reply:
I'm going to resolve this for now so that we can merge the PR. Put this in the backlog